cdap-docs/reference-manual/source/release-notes.rst

```diff
@@ -64,6 +64,8 @@ API Changes
 Known Issues
 ------------
 
+- CDAP has been tested on and supports CDH 4.2.x through CDH 5.3.x. It has not been tested on later versions.
```

Comment:
maybe it's appropriate here to mention that it hasn't been tested on HBase 1.0, which is the reason why it's limited to CDH 5.3 and below.  ("but testing is underway" or similar)